app/models/user.rb

```diff
@@ -20,6 +20,7 @@ class User < ActiveRecord::Base
 
   def unfulfilled_booking_requests
     booking_requests
+      .active
       .includes(:appointment)
       .where(appointments: { booking_request_id: nil })
```

Comment:
It's already within `unfulfilled_booking_requests` which to my mind is perfectly descriptive. Does the addition of the `active` scope change this significantly?